changelog.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.11.0~rc3
+
+* Removes intel microcode (#3955)
```

Comment:
In the final changelog we will squish together all the rc changelog entries (i.e. this line will cancel out `* Added intel-microcode package to include microcode updates (#3913)` below)